manifest_staging/deployment/provider-azure-installer.yaml

```diff
@@ -56,8 +57,13 @@ spec:
               drop:
               - ALL
           volumeMounts:
-            - mountPath: "/provider"
-              name: providervol
+            - name: provider-vol
+              mountPath: /provider
+            - name: mountpoint-dir
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: HostToContainer
+            - mountPath: /etc/kubernetes/custom_environment.json
```

Comment:
This file is only used while testing so I don't think we should be actually adding it to the published yamls?

Reply:
reverted